xfce4-panel pulls in libxfce4ui, so libxfcegui4 needs to go first

--- origsrc/xfce4-genmon-plugin-3.3.0/panel-plugin/Makefile.am	2011-02-22 18:04:42.000000000 -0600
+++ src/xfce4-genmon-plugin-3.3.0/panel-plugin/Makefile.am	2011-10-26 03:14:52.568358800 -0500
@@ -3,12 +3,12 @@ plugin_PROGRAMS = xfce4-genmon-plugin
 
 xfce4_genmon_plugin_CFLAGS =					\
 	-DPACKAGE_LOCALE_DIR=\"$(localedir)\"			\
-	@LIBXFCE4PANEL_CFLAGS@					\
-	@LIBXFCEGUI4_CFLAGS@
+	@LIBXFCEGUI4_CFLAGS@					\
+	@LIBXFCE4PANEL_CFLAGS@
 
 xfce4_genmon_plugin_LDADD =					\
-	@LIBXFCE4PANEL_LIBS@					\
-	@LIBXFCEGUI4_LIBS@
+	@LIBXFCEGUI4_LIBS@					\
+	@LIBXFCE4PANEL_LIBS@
 
 xfce4_genmon_plugin_SOURCES =	\
 	main.c			\
